src/UnitConverter/Unit/Time/Millisecond.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * This file is part of the jordanbrauer/unit-converter PHP package.
+ *
+ * @copyright 2017 Jordan Brauer <jbrauer.inc@gmail.com>
+ * @license MIT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Volume;
+
+use UnitConverter\Unit\{
+    Time\TimeUnit
+};
```

Comment:
Same as Hour.php – Fix namespace, and remove unnecessary import statement for Time\TimeUnit.